Explorer/Assets/Scripts/SceneRunner/Scene/ISceneStateProvider.cs

```diff
@@ -15,4 +17,14 @@ public interface ISceneStateProvider
 
         void SetRunning(SceneEngineStartInfo startInfo);
     }
+
+    public static class SceneStateProviderExtensions
```

Comment:
Why partial? Can you add it into the original file?

Reply:
Same as above, another example is WebControllerRequests